### ISSUE TITLE
Format about labelled vertex graph

### ISSUE THREAD

**Issue by ShunyangLi** [ciaranm/glasgow-subgraph-solver#11]
Hi,

The graph I used is with labelled nodes, no direct, no edge label. So, the CSV format should be:
```
v1, v2,
v3, v3,
v1, l1,
v2, l2,
v3, l3,
v4 l3
```

May I ask Is that format correct?

Thanks

**Comment by ciaranm** [comment]
Not quite. The labels need to be in the third column, so you have to do `v1,,l1` in order to just label a vertex, and you shouldn't include a third column if you're just defining an edge. I think you might also need to remove the whitespace around the commas.

**Comment by ShunyangLi** [comment]
Thanks for answering. One more question.

Is that allowed to execute multiple queries simultaneously?
We run 20 queries for a datagraph. Now we need to load datagraph 20 times. So, if it supports loading multiple queries simultaneously that will be great.

Best

**Comment by ciaranm** [comment]
Currently, no, I've not implemented that. It's probably worth doing: we do some fairly expensive preprocessing on the target graph, which doesn't need to be done more than once. I keep hoping to find a willing victi, er, student volunteer who wants to implement this.

**Comment by ShunyangLi** [comment]
hahaha, we may need to implement this function. If we implement this function, we will send you this code.
By the way, one more question,

0,8  # edge
1,8
4,8
3,8
2,8
0,9
6,9
1,9
7,9
5,10
0,10
1,10
0,,14 # vertex label
1,,14
2,,14
3,,14
4,,14
5,,14
6,,14
7,,14
8,,1000000005
9,,1000000004
10,,1000000003

Does this format correct now?

Thanks.

**Comment by ciaranm** [comment]
Yes, I think that looks right.
